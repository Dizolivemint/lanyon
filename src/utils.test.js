// To allow tests to pass on older platforms: https://travis-ci.org/kvz/lanyon/jobs/200631620#L1079
// require('babel-register')({
//   // This will override `node_modules` ignoring - you can alternatively pass
//   // an array of strings to be explicitly matched or a regex / glob
//   ignore: false,
// })

const utils         = require('./utils')
// const debug      = require('depurar')('sut')
const sut           = utils
const cacheDir      = 'CACHEDIR'
const projectDir    = 'PROJECTDIR'
const lanyonVersion = 'LANYONVERSION'

describe('utils', () => {
  describe('dockerCmd', () => {
    it('should add custom flags', () => {
      const res = sut.dockerCmd({cacheDir, projectDir, lanyonVersion}, 'CMD', 'FLAGS')
      expect(res).toMatchSnapshot()
    })
  })
})
