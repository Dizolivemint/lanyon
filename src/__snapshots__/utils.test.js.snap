exports[`utils dockerCmd should add custom flags 1`] = `"docker run FLAGS --rm --workdir CACHEDIR --user $(id -u) --volume CACHEDIR:CACHEDIR --volume PROJECTDIR:PROJECTDIR kevinvz/lanyon:LANYONVERSION CMD"`;
